### DISCUSSION TITLE
How to multi-cascade a peer-to-peer network

### DISCUSSION THREAD

**Discussion by wd780h** [SoftEtherVPN/SoftEtherVPN#1998] [Q&A]
There are four servers that need to form a peer-to-peer network, but softether seems to only be able to operate in a star network. However, repeated cascading will cause a loop. Can this be solved?

**Comment by shakibamoshiri** [comment]
yes SE designed for hub-spoke topology (start) and not mesh network 
yet you can make it work doing it manually (I tried on linux it was not stable and I gave up finally) 
try other tools if you need a mesh network
